src/_codux/boards/sort-by-button/sort-by-button.board.tsx

```diff
@@ -0,0 +1,8 @@
+import { createBoard } from '@wixc3/react-board';
+import SortByButton from '../../../components/SortByButton/SortByButton';
+
+export default createBoard({
+    name: 'SortByButton',
+    Board: () => <SortByButton />,
+    isSnippet: true,
```

Comment:
Did you open this board to see it?
Btw, this is another reason for why it is great to have a prop named ```className``` that you can give a component and change its styling.

Reply:
You mean the overflow of the text?